test/shm_size.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+  setup_test
+}
+
+function teardown() {
+  cleanup_test
+}
+
+@test "check /dev/shm is changed" {
+  # Start it with the environment variable set, don't use replace_config
+  CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
```

Comment:
looks like you can move this part into `setup()` above.

Reply:
Good point, yay deduplication!